### ISSUE TITLE
each time I run Tortoise (locally), it eats up a lot hard drive space on my C: ?

### ISSUE THREAD

**Issue by embanot** [neonbjb/tortoise-tts#294]
I'm noticing that every time I generate voice clips, my main hardrive space where Windows is installed C: gets lower by around 600-700mb. Keep in mind I'm outputting the voice files to a different hard drive.  At this point its eaten up many gigs of hard drive space.  

Can someone explain why this is happening?  Is there some kind of cache or temp folder that I can clear?

**Comment by I-Have-No-Idea-What-IAmDoing** [comment]
There is a temp/cache folder on the \.cache folder (should be in "C:\Users\YOUR_NAME\.cache\" tortoise uses that for storing the models but it should only be downloaded once.

**Comment by neonbjb** [comment]
Not sure, but if you see anything in the aforementioned cache directory or tortoise that increases in size every time the program is run, please let me know and I'll fix it!

**Comment by embanot** [comment]
yeah so I did some tests and it does looks like something is increasing in size each time I generate a voice clip.   My freespace on C: went from 7.41gb, to 7.25gb to 7.15gb after the last 3 voice clips I generated and doing nothing else.  

Also worth nothing is that while Tortoise in the middle of generating the clip, my my freesapce dropped significantly from 7.25gb to 350mb.  But after it finished,  went back up to 7.15gb, but as you can see still dropped from the 7.25gb. 

Any idea what's going on here?

**Comment by rtphawaii** [comment]
I have the same issue
